src/source/from-shift.js

```diff
@@ -202,7 +202,7 @@ function escapeWTF8(s) {
         if (m == '\u007F') {
             return '\u007F007F';
         }
-        return '\u007F' + m.charCodeAt(0).toString(16);
+        return '\u007F' + m.charCodeAt(0).toString(16).toUpperCase();
```

Comment:
this was necessary to pass roundtrip test which compares AST with escaped WTF-8, because in Rust side we use uppercase for hex string